src/main/java/uk/ac/cam/cl/dtg/isaac/api/managers/GameManager.java

```diff
@@ -1210,17 +1191,18 @@ private IsaacWildcard getWildCardById(final String id) throws ContentManagerExce
             throw new IllegalArgumentException("Error validating filter query.");
         }
 
-        Map<Map.Entry<BooleanOperator, String>, List<String>> fieldsToMatchOutput = Maps.newHashMap();
-
-        // Filter on content tags
-        List<String> tagAnds = Lists.newArrayList();
-        List<String> tagOrs = Lists.newArrayList();
+        List<IContentManager.BooleanSearchClause> fieldsToMatch = Lists.newArrayList();
 
         // handle question categories
         if (null != gameFilter.getQuestionCategories()) {
-            tagAnds.addAll(gameFilter.getQuestionCategories());
+            fieldsToMatch.add(new IContentManager.BooleanSearchClause(
+                    TAGS_FIELDNAME, BooleanOperator.OR, gameFilter.getQuestionCategories()));
```

Comment:
Intentionally switch from AND to OR here as one of our filter queries now search for `problem_solving` or `book` questions